fs/filesystem.go

```diff
@@ -171,6 +186,39 @@ func (fs *FileSystem) mapMacroLookupOptions(file *File) error {
 	return nil
 }
 
+func (fs *FileSystem) scanSeedDirectory(path string) error {
+	return filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+		// If we've encountered an error walking this path, let's return now
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+
+		if filepath.Ext(filepath.Clean(path)) == ".csv" {
+			return fs.recordSeedFile(path)
+		}
+
+		return nil
+	})
+}
+
+func (fs *FileSystem) recordSeedFile(path string) error {
+	name := strings.TrimSuffix(filepath.Base(path), ".csv")
+
+	if prev, found := fs.seeds[name]; found {
+		return fmt.Errorf("%s and %s targets the same model: %s", prev.Path, path, name)
+	}
+	if model := fs.Model(name); model != nil {
+		return fmt.Errorf("model %s and seed %s targets the same model: %s", model.Path, path, name)
+	}
```

Comment:
Returns early if there's a model (`model` from `model.sql`) targeting the same table (`model` from `model.csv`)